README.md

```diff
@@ -16,8 +16,31 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
 ## Table of Contents
 
 * [About jQuery](#about-jquery)
+* [Cache selectors](#cache-selectors)
 
 
 ## About jQuery
 
 [jQuery](http://jquery.com/) is a utility library for easy DOM access & manipulation, event handling, Ajax and more. By using jQuery you can write consise and expressive code which works across modern and legacy browsers. jQuery has extensive tests, detailed documentation, a large active community and an ecosystem of plugins.
+
+## Cache selectors
+### Why?
+To avoid using jQuery every time to look up the element in the DOM you should cache your selectors.
```

Comment:
+ Better performance.
+ You can use descriptive variable names which convey more meaning.